### ISSUE TITLE
Anti-P450scc

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7245]
**Preferred term label:**
Anti-side-chain cleavage enzyme antibody positivity

**Synonyms**
Anti-P450scc

**Definition (free text, please give PubMed ID)**
PMID: 9030872

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:3453

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added, thanks @MaudDLVT
